SU-HSDO/suhumsci
# Feat(STN-771): vertical timeline raised cards

# READY FOR REVIEW

## Summary
Add Timeline classes (`.hb-timeline`, `.hb-timeline__wrapper`, `.ptype-hs-timeline`) to `__raised-cards.scss` stylesheet enabling Timelines to inherit raised card styles.

Update Compatibility Table

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Create a Flexible Page with `hb-raised-cards hb-raised-cards--uniform-height`.
2. Confirm in both themes that the Vertical Timelines receive that raised card treatments.
3. Create a Collection with multiple items (i.e. 2 Timelines and 1 Postcard) to view that they receive the correct `uniform-height` styles.
4. Test Timelines in Wells

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
